<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TvShowSearchWidget</class>
 <widget class="QWidget" name="TvShowSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>606</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hLayoutSearchConfig">
     <item>
      <widget class="QLabel" name="lblScraper">
       <property name="text">
        <string>Scraper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboScraper"/>
     </item>
     <item>
      <widget class="QLabel" name="lblLanguage">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LanguageCombo" name="comboLanguage">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerDropdowns">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MyLineEdit" name="searchString"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lblSuccessMessage">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblErrorMessage">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="results">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Result</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vLayout_Details">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Details to load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabsInfos">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabShowDetails">
          <attribute name="title">
           <string>Show Details</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QGroupBox" name="showInfosGroupBox">
             <property name="title">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="verticalLayout_41">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_421">
                  <item>
                   <widget class="MyCheckBox" name="chkTitle">
                    <property name="text">
                     <string>Title</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkStatus">
                    <property name="text">
                     <string>Status</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkRating">
                    <property name="text">
                     <string>Rating</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkFirstAired">
                    <property name="text">
                     <string>First aired</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkRuntime">
                    <property name="text">
                     <string>Runtime</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkCertification">
                    <property name="text">
                     <string>Certification</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkOverview">
                    <property name="text">
                     <string>Overview</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkGenres">
                    <property name="text">
                     <string>Genres</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkTags">
                    <property name="text">
                     <string>Tags</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkNetwork">
                    <property name="text">
                     <string>Network</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_31">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_43">
                  <item>
                   <widget class="MyCheckBox" name="chkActors">
                    <property name="text">
                     <string>Actors</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkFanart">
                    <property name="text">
                     <string>Fanart</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkPoster">
                    <property name="text">
                     <string>Poster</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkSeasonPoster">
                    <property name="text">
                     <string>Season Poster</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkSeasonFanart">
                    <property name="text">
                     <string>Season Fanart</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkSeasonBanner">
                    <property name="text">
                     <string>Season Banner</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkSeasonThumb">
                    <property name="text">
                     <string>Season Thumb</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkBanner">
                    <property name="text">
                     <string>Banner</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkExtraArts">
                    <property name="text">
                     <string>Artwork</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkThumb">
                    <property name="text">
                     <string>Thumb</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_32">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="line2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chkUnCheckAll">
                <property name="text">
                 <string>(Un)Check all</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblEpisodeOnlyType">
             <property name="text">
              <string>No show details will be loaded because only episodes were selected.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_313">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabEpisodeDetails">
          <attribute name="title">
           <string>Episode Details</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_72">
           <item>
            <widget class="QGroupBox" name="episodeInfosGroupBox">
             <property name="title">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_42">
              <item>
               <layout class="QHBoxLayout" name="verticalLayout_412">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_422">
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeTitle">
                    <property name="text">
                     <string>Title</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeOverview">
                    <property name="text">
                     <string>Overview</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeRating">
                    <property name="text">
                     <string>Rating</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeFirstAired">
                    <property name="text">
                     <string>First aired</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeCertification">
                    <property name="text">
                     <string>Certification</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_312">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_431">
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeDirector">
                    <property name="text">
                     <string>Director</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeWriter">
                    <property name="text">
                     <string>Writer</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeNetwork">
                    <property name="text">
                     <string>Network</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeActors">
                    <property name="text">
                     <string>Actors</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeTags">
                    <property name="text">
                     <string>Tags</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="MyCheckBox" name="chkEpisodeThumbnail">
                    <property name="text">
                     <string>Thumbnail</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_322">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="line3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chkEpisodeUnCheckAll">
                <property name="text">
                 <string>(Un)Check all</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="lblSeasonOrder">
                  <property name="text">
                   <string>Season order</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboSeasonOrder"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblShowOnlyType">
             <property name="text">
              <string>No episodes will be loaded. If you want to load episodes, change the update mode below.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboUpdate">
         <item>
          <property name="text">
           <string>Update TV Show only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Update TV Show and new Episodes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Update TV Show and all Episodes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Update new Episodes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Update all episodes</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">ui/small_widgets/MyCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/small_widgets/MyLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>LanguageCombo</class>
   <extends>QComboBox</extends>
   <header location="global">ui/small_widgets/LanguageCombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
